modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/app/App.js

```diff
@@ -485,8 +485,8 @@ class App extends EventEmitter {
 					this.extractParams(route, path)
 				);
 			})
-			.then(() => nextScreen.evaluateStyles(this.surfaces))
 			.then(() => nextScreen.flip(this.surfaces))
+			.then(() => nextScreen.evaluateStyles(this.surfaces))
```

Comment:
By flipping the screen before evaluating the styles, does that result in significant jank or layout shift when navigating between two pages that have a significant style difference?

I'm not entire sure how you would test this, but if you go from one page with say a white background to another page with a black background, will that delay in evaluating the styles cause a shift?

Reply:
As I tested it along the portal, it behaves better that before. Is there any special use case we should test too?